Jellyfin.Api/Controllers/DisplayPreferencesController.cs

```diff
@@ -182,7 +184,7 @@ public DisplayPreferencesController(IDisplayPreferencesManager displayPreference
                 var order = int.Parse(key.AsSpan().Slice("homesection".Length));
                 if (!Enum.TryParse<HomeSectionType>(displayPreferences.CustomPrefs[key], true, out var type))
                 {
-                    type = order < 7 ? defaults[order] : HomeSectionType.None;
+                    type = order < 8 ? defaults[order] : HomeSectionType.None;
```

Comment:
I think this line doesn't make any sense anymore because there can only be 7 homesections in total. Same with the last item in the defaults list.